native/sendctrlc/sendctrlc.vcxproj

```diff
@@ -142,6 +145,7 @@
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <PreprocessorDefinitions>WIN32;NDEBUG;_CONSOLE;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
```

Comment:
this app is actually single-threaded IIRC, but should not make much difference. Will double check

Reply:
By default "Runtime Library" is set to "Multi-threaded Debug DLL (/MDd)" for Debug and "Multi-threaded DLL (/MDd)".

Looks like there is no "single-threaded" option:
![image](https://user-images.githubusercontent.com/607109/48863352-1c678580-edda-11e8-8fbc-1069ad3a5b60.png)